src/data/links.yaml

```
@@ -1750,3 +1750,39 @@ links:
       Wish you could avoid airport queues, tedious transfers and long drives?
       Want to travel sustainably and enjoyably by train instead - to fantastic accommodation in the best ski resorts in the Alps?
       Snowcarbon is an independent guide founded by ski journalist Daniel Elkan to help you do exactly that.
+
+  - title: Runamics Running & Sports Gear
+    url: https://www.runamics.com
+    countries: []
+    categories: [clothing]
+    tags: [clothing]
+    description: Crowd-funded for-purpose eco-friendly alternatives to traditional sports gear.  We combine merino wool with wood-based cellulosic fibres like Lyocell. We make experiments with highly innovative fibres like biodegradable elastic fibres or “polyester-like” substitutes.  We're a climate neutral company- We avoid emissions where we can and where they can't be avoided, we compensate them. We will repair your runamics garment and send it back to you for free or discount a replacement. We use plastic-free packaging and ship globally with DHL GoGreen.
```

Can you use the multiline YAML syntax like the above links?

```suggestion
    description: >
      Crowd-funded for-purpose eco-friendly alternatives to traditional sports gear. 
      We combine merino wool with wood-based cellulosic fibres like Lyocell. We make 
      experiments with highly innovative fibres like biodegradable elastic fibres or “polyester-like”
      substitutes.  WE're climate neutral always working to reduce our footprint, use plastic-free packaging and ship globally with DHL GoGreen.
```

Also GO GREEN WHAT IS THIS :D

---

[DHL GoGreen](https://www.dpdhl.com/en/responsibility/environment-and-solutions/gogreen-program.html) is news to me.  The link outlines their sustainability goals in impressive detail.  Would be good to add.